components/gitpod-db/go/workspace.go

```diff
@@ -18,12 +18,13 @@ import (
 
 // Workspace represents the underlying DB object
 type Workspace struct {
-	ID          string         `gorm:"primary_key;column:id;type:char;size:36;" json:"id"`
-	OwnerID     uuid.UUID      `gorm:"column:ownerId;type:char;size:36;" json:"ownerId"`
-	ProjectID   sql.NullString `gorm:"column:projectId;type:char;size:36;" json:"projectId"`
-	Description string         `gorm:"column:description;type:varchar;size:255;" json:"description"`
-	Type        WorkspaceType  `gorm:"column:type;type:char;size:16;default:regular;" json:"type"`
-	CloneURL    string         `gorm:"column:cloneURL;type:varchar;size:255;" json:"cloneURL"`
+	ID             string         `gorm:"primary_key;column:id;type:char;size:36;" json:"id"`
+	OrganizationId uuid.UUID      `gorm:"column:organizationId;type:char;size:36;" json:"organizationId"`
```

Comment:
If it can be null, we may want to encode it as `*uuid.UUID` else the default value would be `uuid.Nil` - the zero value of a UUID